.github/workflows/generate.yml

```diff
@@ -35,6 +35,10 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
+      - name: Test generate script
```

Comment:
I like adding tests here, I would prefer that we create a third workflow instead of adding it to the existing workflows

Reply:
Good idea - moved it into its own workflow - https://github.com/ethereum-optimism/ethereum-optimism.github.io/runs/7902324372?check_suite_focus=true